src/tiled/scriptdialog.cpp

```diff
@@ -267,6 +291,21 @@ ScriptDialog::NewRowMode ScriptDialog::newRowMode() const
     return m_newRowMode;
 }
 
+ScriptButtonGroup *ScriptDialog::addRadioButtonGroup(const QString &labelText,
+                                                     const QStringList &values,
+                                                     const QString &toolTip,
+                                                     const QStringList &buttonToolTips)
+{
+    QGroupBox *groupParent = new QGroupBox(this);
+    QHBoxLayout *hBox = new QHBoxLayout(groupParent);
+    ScriptButtonGroup *buttonGroup = new ScriptButtonGroup(groupParent, hBox);
+    buttonGroup->addItems(values, buttonToolTips);
+    hBox->addStretch(rightColumnStretch);
+    groupParent->setLayout(hBox);
```

Comment:
Ah, you can remove this line because `new QHBoxLayout(groupParent)` already assigned the layout to that widget.